apps/condo/domains/user/components/UserOrganizationsList.tsx

```diff
@@ -48,6 +48,7 @@ interface IOrganizationEmployeeItem {
 const OrganizationEmployeeItem: React.FC<IOrganizationEmployeeItem> = (props) => {
     const intl = useIntl()
     const OrganizationMessage = intl.formatMessage({ id: 'pages.condo.property.field.Organization' })
+    const InnMessage = intl.formatMessage({ id: 'pages.condo.property.field.Inn'})
```

Comment:
In our domain notation we use word `tin` instead of `inn`:

<img width="739" alt="image" src="https://github.com/open-condo-software/condo/assets/50069872/fb737202-6ceb-44dd-8299-0c2733ed1ed0">

Look, you've already got your answer few lines below :)